tests/nested/manual/fde-on-classic/task.yaml

```diff
@@ -126,8 +126,8 @@ execute: |
   # Test aborting a kernel installation before a reboot
   refresh_rebooting_snap pc-kernel-new.snap pc-kernel no-reboot
   remote.exec sudo snap abort --last=install
-  # Make sure everything is in the same state
-  not remote.exec stat /run/mnt/ubuntu-boot/EFI/ubuntu/try-kernel.efi
+  # Make sure everything is in the same state (might need a retry)
```

Comment:
maybe this should `snap watch --last=install` for the change to complete before checking?

Reply:
Isn't that wait already happenning in the call to `snap abort`? Or for that command that is not really the case?